lib/Transforms/CMakeLists.txt

```diff
@@ -3,8 +3,8 @@ add_subdirectory(Utils)
 add_llvm_library(MLIRTransforms
   Canonicalizer.cpp
   CSE.cpp
+  DataCopy.cpp
```

Comment:
DataCopy sounds a like a generic library utility function. How about DataCopyGeneration or DataTransferGeneration?

Reply:
Yes, DataCopyGeneration or DataTransferGeneration are better names, albeit longer. Also, I missed mentioning that "array packing" is another name used in the literature for this optimization:
http://www.cs.utexas.edu/users/flame/pubs/TOMS-BLIS-Analytical.pdf

"MemrefPacking" (-affine-memref-pack) in this context is another possibility.

Reply:
Changed to AffineDataCopyGeneration for now.